obspy/io/sac/sactrace.py

```diff
@@ -1100,7 +1100,7 @@ def reftime(self, new_reftime):
     # --------------------------- I/O METHODS ---------------------------------
     @classmethod
     def read(cls, source, headonly=False, ascii=False, byteorder=None,
-             checksize=False, debug_strings=False):
+             checksize=False, debug_strings=False, **kwargs):
```

Comment:
Instead of using `kwargs` - can you please directly specify the argument name in the function definition? This is IMHO much easier to read and understand.